app/auth.go

```diff
@@ -295,15 +322,23 @@ mainloop:
 }
 
 // Stops the running MenderAuthManager. Must not be called in the same go
-// routine as Run().
+// routine as run(). This is idempotent, it is safe to call on a stopped
```

Comment:
Nit:
`This is idempotent, _and_ is safe to call on a stopped service`. right?

Reply:
Being idempotent means that it is safe to call on a stopped service, so I think the original sentence is better. They are not two different properties, just the same property listed, and then explained.

Reply:
No, being idempotent means that it doesn't change when applied multiple times no? It's not specific to a state of the environment the function is called on.

Reply:
But it's not important.

Reply:
Hehe true, I didn't mean that this is the definition of idempotent, but that this is the effect of applying it in this particular case. But yes, not really important.